scaling_performance/install_practices.adoc

```diff
@@ -81,6 +81,12 @@ timeout = 10
 <1> 20 forks is ideal, because larger forks can lead to installations failing.
 <2> Pipelining reduces the number of connections between control and target nodes, helping to improve installer performance.
 
+[NOTE]
+====
+By default, logging is disabled in Ansible. Ensure logging in the
+`/etc/ansible/ansible.cfg` file is marked as visible.
+====
+
```

Comment:
@bmcelvee What does logging is marked as visible mean? That is an odd phrase to me. How do you set to visible?